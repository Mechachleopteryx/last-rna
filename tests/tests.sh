#! /bin/sh

cd $(dirname $0)

PATH=..:$PATH

{
    rna-alignment-stats --help
    rna-alignment-stats test.psl
} | diff -u tests.out -
