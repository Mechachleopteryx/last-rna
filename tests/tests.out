Usage: rna-alignment-stats [options] alignments.psl

Print a table of summary statistics for RNA-to-genome alignments.

Options:
  -h, --help  show this help message and exit
AJ238554.1-8	749	736	0	0	6	2	0
NM_012460.2-4	1108	817	221	36	16	626	0
NM_145280.4-318	4748	1736	4	226	2630	284	0
NR_027349.1-2	927	727	32	79	71	8	0
NM_183043.2-6	3339	2234	263	652	89	1708	0
U02629.1-17	697	611	9	2	44	22	0
BC105276.1-48	470	440	0	29	1	8	0
AB384604.1-3	4094	3910	5	8	38	186	0
AK027129.1-127	1493	1283	4	54	54	19	0
AK090512.1-929	2239	1215	514	21	144	173	0
AK098785.1-9	3118	837	1387	736	23	18	0
NM_000211.4-7	2973	396	1	2568	5	11	0
NM_016497.3-4	690	609	56	13	6	26	0
AK124587.1-824	4668	334	0	4214	51	24	0
NM_000993.4-45	524	443	60	16	3	4	0
NR_034162.1-230	3489	1033	1909	251	41	42	0
AK293331.1-3	610	577	2	0	16	6	0
E07591.1-40	2033	332	967	646	22	7	0
NM_021019.4-18	827	661	96	21	45	1	0
AK315413.1-12	490	488	0	0	2	261	0
NM_152772.2-4	2255	1054	344	478	84	36	0
NR_040021.1-693	2112	955	281	645	77	100	0
NM_001100170.1-2	2985	2626	308	16	16	2	0
NM_006280.2-2	727	502	95	103	12	2	0
AK097770.1-498	1781	712	276	472	95	47	0
AB209062.1-386	4230	548	2111	1525	37	42	0
AK094875.1-2977	1756	918	316	73	119	35	0
NM_001282233.1-4	996	321	637	22	11	3	0
NM_004891.3-2	528	443	34	10	16	2	0
NM_000980.3-14	671	568	43	13	13	3	0
NM_006870.3-14	1614	1377	150	20	23	9	0
AK130965.1-79	538	523	2	0	6	2	0
